tool/build_release.sh

```diff
@@ -15,18 +15,28 @@ DEVTOOLS_DIR="${TOOL_DIR}/.."
 
 pushd $TOOL_DIR
 
-# Use the Flutter SDK from flutter-sdk/.
-FLUTTER_DIR="`pwd`/flutter-sdk"
-PATH="$FLUTTER_DIR/bin":$PATH
-
-# Make sure the flutter sdk is on the correct branch.
-./update_flutter_sdk.sh
+if [[ $1 = "--no-update-flutter" ]]
+then
+  # Use the Flutter SDK that is already on the user's PATH.
+  FLUTTER_EXE=`which flutter`
+  echo "Using the Flutter SDK that is already on PATH: $FLUTTER_EXE"
+else
+  # Use the Flutter SDK from flutter-sdk/.
+  FLUTTER_DIR="`pwd`/flutter-sdk"
+  PATH="$FLUTTER_DIR/bin":$PATH
+
+  # Make sure the flutter sdk is on the correct branch.
+  ./update_flutter_sdk.sh
+fi
 
 popd
 
 # echo on
 set -ex
 
+which flutter
+flutter --version
```

Comment:
If these are meant to be here, then adding a bit of context to the messages could be helpful:
```suggestion
echo "Flutter Path: $(which flutter)"
echo "Flutter Version: $(flutter --version)"
```